gradle/plugins/src/main/kotlin/com.hedera.gradle.spotless.gradle.kts

```diff
@@ -21,7 +21,7 @@ spotless {
     isEnforceCheck = false
 
     // optional: limit format enforcement to just the files changed by this feature branch
-    ratchetFrom("origin/develop")
+    ratchetFrom("origin/main")
```

Comment:
The file modified here no longer exists. For this change, you now need to delete this file:
https://github.com/hashgraph/hedera-services/blob/develop/gradle/development-branch.txt

If `development-branch.txt` does not exist, we defaults to `main`.